lib/kvdb/ikvdb.c

```diff
@@ -555,15 +548,15 @@ ikvdb_storage_add(const char *kvdb_home, struct kvdb_cparams *params)
             char buf[PATH_MAX];
 
             static_assert(sizeof(buf) == sizeof(params->storage.mclass[HSE_MCLASS_BASE].path),
-                          "mismatched buffer sizes");
+                    "mismatched buffer sizes");
 
             if (meta.km_storage[i].path[0] != '\0') {
                 err = merr(EEXIST);
                 goto errout;
             }
 
-            err = kvdb_home_storage_path_get(kvdb_home, params->storage.mclass[i].path,
-                                             buf, sizeof(buf));
+            err = kvdb_home_storage_path_get(
```

Comment:
Another point of discussion.

Reply:
What's the concern?

Reply:
Putting as many parameters on the same line as the function name or starting them on the next line.

```c
func(
  param1, ..., param2);

vs

func(param1, ...,
  param2);
```

Reply:
The current config attempts these three layouts in order:
```
                                                                                   |<-- column limit
foobar_blah_blah_yoyo(param1, param2, param3);

foobar_blah_blah_yoyo(
    param1, param2, param3, param4, param5);

foobar_blah_blah_yoyo(
    param1,
    param2,
    param3,
    param4,
    param5,
    param6);
```

Reply:
I see. Is version 3 what we want in that case versus packing as many args into a line as possible, overflowing, and repeating?

Reply:
All three in the given order is right.  It should never use multiple lines when one would do...